src/store/sagas/wallet.ts

```diff
@@ -182,6 +182,20 @@ export function* signAndSend(wallet: WalletAdapter, tx: Transaction): SagaGenera
   const signature = yield* call([connection, connection.sendRawTransaction], signedTx.serialize())
   return signature
 }
+export function* signAllTransaction(
```

Comment:
WalletAdapter already has 'signAllTransactions' method, look at its docs

Reply:
ok, but this method not work